DOL.WHD.Section14c.Log/LogHelper/BaseApiController.cs

```diff
@@ -9,50 +9,63 @@
         }
 
-        protected BadRequestResult BadRequest(string message)
+        /// <summary>
+        /// Throw Bad Request Exception
+        /// </summary>
+        /// <param name="message"></param>
+        protected void BadRequest(string message)
         {
-            throw new ApiBusinessException((int)HttpStatusCode.BadRequest, message, HttpStatusCode.BadRequest, "");
-            return null;
+            throw new ApiBusinessException((int)HttpStatusCode.BadRequest, message, HttpStatusCode.BadRequest);
         }
 
-        protected BadRequestResult BadRequest(string message, Exception ex)
+        /// <summary>
+        /// Throw Conflict Exception
+        /// </summary>
+        /// <param name="message"></param>
+        protected void Conflict(string message)
         {
-            throw new ApiBusinessException((int)HttpStatusCode.BadRequest, ex.Message, HttpStatusCode.BadRequest, "");
-
-            return null;
-        }
-        
-        protected ConflictResult Conflict(string message)
-        {
-            throw new ApiDataException((int)HttpStatusCode.Conflict, message, HttpStatusCode.Conflict, "");
-            return null;
+            throw new ApiDataException((int)HttpStatusCode.Conflict, message, HttpStatusCode.Conflict);
         }
-        
-        protected InternalServerErrorResult InternalServerError(string message)
+
+        /// <summary>
+        /// Throw Internal Server Error Exception
+        /// </summary>
+        /// <param name="message"></param>
+        protected void InternalServerError(string message)
         {
-            throw new ApiException((int)HttpStatusCode.InternalServerError, message, HttpStatusCode.InternalServerError, "");
-            return null;
+            throw new ApiException((int)HttpStatusCode.InternalServerError, message, HttpStatusCode.InternalServerError);
         }
 
-        protected UnauthorizedResult Unauthorized(string message)
+        /// <summary>
+        /// Throw Unauthorized Exception
+        /// </summary>
+        /// <param name="message"></param>
+        protected void Unauthorized(string message)
         {
-            throw new ApiDataException((int)HttpStatusCode.Unauthorized, message, HttpStatusCode.Unauthorized, "");
-
-            return null;
+            throw new ApiDataException((int)HttpStatusCode.Unauthorized, message, HttpStatusCode.Unauthorized);
         }
 
-        protected ExceptionResult ExpectationFailed(string message)
+        /// <summary>
+        /// Throw ExpectationFailed Exception
+        /// </summary>
+        /// <param name="message"></param>
+        protected void ExpectationFailed(string message)
         {
-            throw new ApiDataException((int)HttpStatusCode.ExpectationFailed, message, HttpStatusCode.ExpectationFailed, "");
-            return null;
+            throw new ApiDataException((int)HttpStatusCode.ExpectationFailed, message, HttpStatusCode.ExpectationFailed);
         }
     }
 }
```

Comment:
I realized this morning I probably made the wrong suggestion on these.  Rather than keep blocking this PR, I created chore issue #327.  Thanks for making these changes, @dciceman!